tests/ert_tests/status/test_tracking_integration.py

```diff
@@ -233,3 +242,106 @@ def test_tracking(
                     )
         thread.join()
     FeatureToggling.reset()
+
+
+def run_sim(start_date):
```

Comment:
Does this have to be a function or could we just inline it in `test_tracking_time_map`?

Reply:
Call it twice, one for refcase, and one for the forward model, but could have it defined inside the function if you think that is better?

Reply:
Ah, I thought it was just called once so it's fine.
To bike-shed a bit, I suggest changing docstring to "Create a summary file, the contents of which are not important."